sympy/crypto/crypto.py

```diff
@@ -349,6 +398,53 @@ def decipher_affine(msg, key, symbols=None):
     """
     return encipher_affine(msg, key, symbols, _inverse=True)
 
+def encipher_atbash(msg, symbols=None):
+    r"""
+    Enciphers a given ```msg``` into its Atbash ciphertext and returns it.
+
+    Notes
+    =====
+	
```

Comment:
The tests failing. I see some trailing spaces to be removed here

Reply:
Oops, sorry.My bad, didn't run the test_code_quality.py before commiting. Will fix it immediately. N++ automatically indents trailing spaces with newlines, didn't notice this time.